api/src/controllers/sign-up-controller.js

```diff
@@ -0,0 +1,43 @@
+import usersService from '../services/users-service';
+
+export default class setUpThisController{
+
+    static register(app){
+        
+        app.get('/signup', function(req, res) {
+            // usersService.signUpUser(function(newUser){
+                let model = {
+                    // user : newUser
+                };
+                res.render('signup', model);
+            // });
+        });
+    
+        app.post('/signup', function(req, res) {
+            // TODO: signup code
+    
+            let newUserName = req.body.userName
```

Comment:
Use object destructuring here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Ask me if youre not sure what this means